php-practice_1.php

```diff
@@ -0,0 +1,196 @@
+<?php
+// Q1 変数と文字列
+
+ $message = '私の名前は「伊東」です。';
+ echo $message;
+ 
+?>
+
+// Q2 四則演算
+<?php 
+
+ $num = 5*4;
+ 
+ echo $num . "\n";
+ 
+ echo $num / 2;
+ 
+?>
+
+// Q3 日付操作
+<?php 
+
+ echo date("現在時刻は、Y年m月d日 H時i分s秒です。")
+
+?>
+
+// Q4 条件分岐-1 if文
+<?php
+$device = 'mac';
+
+if ($device === 'windows' || $device === 'mac') {
+    echo"使用OSは、{$device}です。";
```

Comment:
[nits]
echo の後は空白を開けてください！

コーディング規約を守ることで、可読性が高まります！
```suggestion
    echo "使用OSは、{$device}です。";
```

Reply:
空白開けました****